source/usage-examples/insertMany.txt

```diff
@@ -30,7 +30,7 @@ Click `here <{+example+}/insertMany.go>`__ to see a fully runnable example.
 Expected Result
 ---------------
 
-After you run the code example, two new documents will be present in the
+After you run the preceding example, two new documents will be present in the
```

Comment:
This mixes present tense ("After you run the preceding example") and continuous future tense ("documents will be present") in a sentence which makes it less clear.
The style guide suggests only using present tense: https://docs.mongodb.com/meta/style-guide/writing/use-present-tense/

A couple ways you can reword the second phrase which seem to match up with how the other usage examples are worded:
- "your collection should contain two new documents..."
- "you can find two new documents..."

Applies to all instances of future tense.